### PR TITLE
Fix/umb build

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
Thi PR fixes the export issue in the umd build for the browser

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +22,7 @@
     "docs:watch": "watchy -w lib npm run docs:dev",
     "docs:publish": "npm run docs:build && ./node_modules/contentful-sdk-jsdoc/bin/publish-docs.sh contentful.js contentful",
     "lint": "eslint lib test",
-    "pretest": "npm run lint",
+    "pretest": "standard lib/*.js && standard lib/**/*.js && standard test/**/*.js",
```
Comment (axe312ger) [inline package.json]: nope nope, please do not remove eslint again

File: package.json [hunk 2]
```diff
@@ -67,6 +67,7 @@
     "babel-loader": "^6.2.4",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-rewire": "^1.0.0",
+    "babel-plugin-transform-es2015-modules-umd": "^6.24.0",
```
Comment (axe312ger) [inline package.json]: not used also

File: version.js [hunk 3]
```diff
@@ -1 +1 @@
-module.exports = ''
+module.exports = '4.1.0'
```
Comment (axe312ger) [inline version.js]: sure about this?

Comment (Khaledgarbaya) [inline version.js]: this is automated so it does not matter

**Pull request merged**